tests/func/test_init.py

```diff
@@ -66,7 +66,7 @@ def test_cli(self):
         self._test_init()
 
 
-def test_init_quiet_should_not_display_welcome_screen(git, caplog):
+def test_init_quiet_should_not_display_welcome_screen(scm, caplog):
```

Comment:
This remind me of iterative/dvc.org/issues/540 though.

Reply:
@jorgeorpinel Yeah, me too. Maybe we will rename it to git after the migration is done.

Reply:
We probably won't. People will assume `git` being some gitpython instance.